.goreleaser.yml

```diff
@@ -24,6 +24,7 @@ archives:
       linux: Linux
       windows: Windows
       amd64: x86_64
+    name_template:  "{{ .ProjectName }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}_{{ .Mips }}{{ end }}"
```

Comment:
so nice GH wont let me comment on the line that isn't changed in the PR...

could you remove "windows" from `goos` key? https://github.com/ForgeRock/forgeops-cli/pull/46/files#diff-42e26dc67aed8aa3edb2472b4403288c1699fb6dc47419b9a475f0f224fe4689R15

Reply:
and that link doesn't work either ~Loc 25

Reply:
I think we should also remove `arm` from goarch. I don't think we're going to see much arm7 action with this tool